.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+---
+version: 2
+jobs:
+  build:
+    machine:
+      image: ubuntu-1604:201903-01
+    working_directory: /tmp/build
+    environment:
+      DOCKER_USERNAME: travisciresin
+      DOCKER_IMAGE: balena/open-balena-base
+      DOCKER_BRANCH_TAG: "${CIRCLE_BRANCH//[^a-zA-Z0-9_-]/-}"
+    steps:
+      - checkout
+      - run:
+          name: Build
+          command: |
+            docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
```

Comment:
@mikesimos I enabled circleCI in order to publish no-systemd variant, but apparently miss password configuration. Login fails during the build. Would take a look here?

Reply:
@roman-mazur done.

Reply:
Was it a configuration on CircleCI admin side?

Reply:
Yes

Reply:
@roman-mazur resolving this